### DISCUSSION TITLE
Is it possible to create an eap with computer vision sdk?

### DISCUSSION THREAD

**Discussion by thomedw** [AxisCommunications/acap-computer-vision-sdk-examples#145] [Q&A] [answered]
Hi,

I am wondering whether it is possible to use something like `acap-build` with computer vision sdk.

https://github.com/AxisCommunications/acap-native-sdk-examples/blob/a1c57eb3e12820841c769d62e37094a92607f411/hello-world/Dockerfile#L12

Thank you.

**Comment by Corallo** [comment, marked as answer]
Hello @thomedw 

That is not the intendend way to use the CV SDK.
The `acap-build` script is built to simplify the building and linking to the device API.
Since in the CV SDK you use a container, you don't need the `acap-build`, you don't have access to the axis API anyway directly, except for the vdo library that can be used via opencv.

You can see here and example of how to build your [own application](https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/blob/main/hello-world-cross-compiled/Dockerfile).
